recipe/meta.yaml

```diff
@@ -71,17 +68,20 @@ requirements:
     - setuptools
     - cython
     - {{ mpi }}  # [mpi != 'nompi']
-    - tempest-remap  # [tempest == 'tempest']
+    - tempest-remap 2.0.2  # [tempest == 'tempest']
     - libnetcdf * {{ mpi_prefix }}_*  # [tempest == 'tempest']
+    - eigen  # [tempest == 'tempest']
+    - netcdf-cxx-legacy  # [tempest == 'tempest']
```

Comment:
Newly added to conda-forge and not compatible with `netcdf-cxx4`.  The newest version of `tempest-remap` doesn't need this package, so we should drop it in new builds.